### PR TITLE
Added vulnerability reject

### PR DESCRIPTION
**Pull request by Cheshiriks**
Снимок

### CODE REVIEW

File: save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt [hunk 1]
```diff
@@ -221,6 +221,23 @@ class VulnerabilityController(
...
+    @RequiresAuthorizationSourceHeader
+    @PreAuthorize("hasRole('ROLE_SUPER_ADMIN')")
+    fun reject(
+        @RequestBody vulnerabilityDto: VulnerabilityDto,
+    ): Mono<StringResponse> = blockingToMono {
+        vulnerabilityService.reject(vulnerabilityDto)
+    }.map {
+        ResponseEntity.ok("Vulnerability was successfully rejected")
+    }
+
```
Comment (sanyavertolet) [inline save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt:240]: To my mind, this method needs `@PreAuthorize("hasRole('ROLE_SUPER_ADMIN')")` annotation

**Review by sanyavertolet — APPROVED** [review]
lgtm

**Comment by akuleshov7** [comment]
@Cheshiriks can you please give me a chance to merge?

**Comment by akuleshov7** [comment]
test

**Pull request merged**